docs/configure-scheduler.rst

```diff
@@ -0,0 +1,25 @@
+Configure scheduler type
+------------------------
+
+If a custom scheduler type is required (for example if you are deploying with Portworx or another storage provider), this can be set on either each nodepool:
```

Comment:
Doesn't sound quite right. 
Maybe instead "can be set on either each nodepool" > "can either be set on the cluster and / or on each nodepool"